hw/ip/keymgr/data/keymgr_testplan.hjson

```diff
@@ -113,8 +133,10 @@
 
             Stimulus:
             Force one of these or both
-            - force advance, gen_id and gen_out to not one-hot
-            - force KMAC FSM to default branch to trigger FSM error
+            - Force advance, gen_id and gen_out to not one-hot
+              - Path: `dut.u_ctrl.adv_en_o/id_en_o/gen_en_o`
+            - Force KMAC FSM to default branch to trigger FSM error
+              - Path: `dut.u_kmac_if.state_d`
```

Comment:
@tjaychen please confirm if these paths are correct, which I will force them to trigger invalid_cmd error.

Reply:
this looks good!